observations/README.md

```diff
@@ -0,0 +1,1279 @@
+
+<details>
+<summary>Example JSON</summary>
+
+```JSON
+
+{
+    "resource": {
+        "resourceType": "Observation",
+        "id": "C45E3DA5-D7BC-4FDF-B4E0-6CCECCE60D24",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"
+            ]
+        },
+        "identifier": [
+            {
+                "system": "https://PSSAdaptor/D5445",
+                "value": "C45E3DA5-D7BC-4FDF-B4E0-6CCECCE60D24"
+            }
+        ],
+        "status": "final",
+        "code": {
+            "coding": [
+                {
+                    "extension": [
+                        {
+                            "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
+                            "extension": [
+                                {
+                                    "url": "descriptionDisplay",
+                                    "valueString": "Serum cholesterol"
+                                }
+                            ]
+                        }
+                    ],
+                    "system": "http://snomed.info/sct",
+                    "code": "1005671000000105",
+                    "display": "Serum cholesterol level"
+                }
+            ],
+            "text": "Serum cholesterol"
+        },
+        "subject": {
+            "reference": "Patient/80bd1375-a1e3-4137-95ff-fb3316cf3496"
+        },
+        "effectiveDateTime": "2001-03-30",
+        "issued": "2010-02-09T12:31:51.000+00:00",
+        "performer": [
+            {
+                "reference": "Practitioner/C5DEFBF3-0174-BC6F-182C-B777B9C6FF43"
+            }
+        ],
+        "valueQuantity": {
+            "value": 6.1,
+            "unit": "millimole per liter",
+            "code": "mmol/L"
+        },
+        "related": [
+            {
+                "type": "derived-from",
+                "target": {
+                    "reference": "Observation/31465FC9-15B2-4391-A5EF-8F70C2154AAF"
+                }
+            }
+        ]
+    }
+}
+```
+
+</details>
+
+# JSON FHIR > XML HL7
+
+FHIR Observations can be mapped to the following HL7 components. The mapping for each GP Connect 
+observation type (in **bold**) is described in the sections below:
+
+- ObservationStatement (**[Uncategorised Data](#Uncategorised-Data-(JSON-FHIR->-XML-HL7))** and **[Test Result](#Test-Result-(JSON-FHIR->-XML-HL7))**)
+- CompoundStatement (**[Blood Pressure](#Blood-Pressure-(JSON-FHIR->-XML-HL7))**, **[Test Group Header](#Test-Group-Header-(JSON-FHIR->-XML-HL7))**)
+- NarrativeStatement (**[Comment Note](#Comment-Note-(JSON-FHIR->-XML-HL7))** / **[Filing Comment](#Filing-Comment-(JSON-FHIR->-XML-HL7))**)
+
+## Uncategorised Data (JSON FHIR > XML HL7)
+
+Uncategorised data is mapped from an FHIR `Observation` to a HL7 `ObservationStatement` where it is not deemed to be a 
+Blood Pressure or Comment Note. 
```

Comment:
Worth linking to the Blood Pressure / Comment Note sections below?

Reply:
yes